Dockerfile-slim.template

```diff
@@ -1,6 +1,7 @@
 FROM buildpack-deps:jessie-curl
 
-RUN groupadd -r node && useradd -r -g node node
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --create-home node
```

Comment:
Maybe the chance of that is unlikely, but since the possibility is still there, why not be on the safe side?
Without that check no error will surface and hidden errors are hard to debug.

In any case, there is absolutely no drawback in having that check, why advocate for a solution which has no benefits other than it's a shorter command line?